code/game/objects/structures/votingbox.dm

```diff
@@ -178,24 +178,28 @@
 	for(var/option in results)
 		tally += "<li>\"<div class='content'>[option]</div>\" - [results[option]] Vote[results[option] > 1 ? "s" : ""].</li>"
 	tally += "</ol>"
-	P.extra_headers = {"
-	<meta http-equiv='X-UA-Compatible' content='IE=edge'/>
-	<style>
-		.content{
-			max-width:250px;
-			display:inline-block;
-			overflow:hidden;
-			text-overflow:ellipsis;
-			white-space:nowrap;
-			vertical-align:bottom
-		}
-		.content br {
-			display: none;
-		}
-		.content hr {
-			display: none;
-		}
-	</style>"}
+	/*
```

Comment:
Delete, don't comment out please

Reply:
removed this already

Reply:
actually, I put the style in.  Gave me an idea on how I can fix the form input.